test_scripts/Defects/4_6/1218_Mandatory_parameter_spaceAvailable_is_not_provided_in_response_for_PutFile_request.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1218
+--
+-- Precondition:
+-- 1) SDL is built with EXTERNAL_PROPRIETARY flag.
+-- 2) SDL and HMI are started. First ignition cycle.
+-- 3) Connect device
+-- 4) Register new application
+
+-- Description:
+-- Steps to reproduce:
+-- 1) App sends valid "PutFile" request
+-- Expected:
+-- 1) Successful process PutFile{ success = true, resultCode = "SUCCESS", spaceAvailable=xxx}.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Defects/commonDefects')
+
+--[[ Local Functions ]]
+local function putFileSuCCESS(self)
+  local paramsSend = {
+    syncFileName = "icon.png",
+    fileType = "GRAPHIC_PNG"
+  }
+  local cid = self.mobileSession1:SendRPC( "PutFile", paramsSend, "files/icon.png")
+  self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+  :ValidIf(function(_, data)
+  	if data.payload.spaceAvailable ~= nil then
+  		return true
+  	else
+  		return false
```

Comment:
add error message in case of error, like `return false, "PutFile response does not contain spaceAvailable parameter." 
and also please squash commits.